src/app/modules/perinatal/components/listado-perinatal.component.ts

```diff
@@ -1,10 +1,14 @@
 import { CarnetPerinatalService } from './../services/carnet-perinatal.service';
+import { OrganizacionService } from '../../../services/organizacion.service';
 import { Component, OnInit, Input } from '@angular/core';
 import { Location } from '@angular/common';
 import { Router } from '@angular/router';
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
 import { Auth } from '@andes/auth';
+import { IOrganizacion } from 'src/app/interfaces/IOrganizacion';
+import { ProfesionalService } from 'src/app/services/profesional.service';
+import { Console } from 'console';
```

Comment:
IOrganizacion y Console se importan pero no se utilizan
```suggestion
import { ProfesionalService } from 'src/app/services/profesional.service';
```